app/controllers/commands_controller.rb

```diff
@@ -5,6 +5,7 @@ class CommandsController < ApplicationController
   # GET /commands.json
   def index
     @commands = Command.all
+    'tttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt'
```

Comment:
INFO: Metrics/LineLength: Line is too long. [152/120] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)